index.js

```diff
@@ -1,4 +1,4 @@
-import WebView from './lib/WebView';
+import WebView from './src/WebView';
```

Comment:
Why is this changed?

Reply:
Because it complained on compile for me. You are right though, that should not be part of this PR. Will revert.